bin/plugin/commands/test-plugins.js

```diff
@@ -0,0 +1,323 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs-extra' );
+const { execSync, spawnSync } = require( 'node:child_process' );
```

Comment:
```suggestion
const { execSync, spawnSync } = require( 'child_process' );
```
When i run command in local environment it show me `Error: Cannot find module 'node:child_process'` error. After applying the changes it working for me.